eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -201,51 +201,43 @@ stages:
-                            ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
-                            Target: MavenCentral
-                            BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
-                            JavaRepoRoot: $(Pipeline.Workspace)/azure-sdk-for-java
-                        - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
-                          parameters:
-                            ArtifactName: ${{parameters.ArtifactName}}-${{artifact.name}}-mavencentral-$(System.JobAttempt)
-                            ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
-                      # After publishing to Maven, publish to the azure-sdk-for-java feed. The reason for
-                      # this is that the azure-sdk-for-java feed will have the package available immediately
-                      # whereas Maven can take several hours for the package to become available.
-                      - template: /eng/pipelines/templates/steps/java-publishing.yml
-                        parameters:
-                          ArtifactID: ${{artifact.name}}
-                          GroupID: ${{artifact.groupId}}
-                          ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
-                          RepositoryUrl: https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-java/maven/v1
-                          Target: JavaDevFeed
-                          BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
-                          JavaRepoRoot: $(Pipeline.Workspace)/azure-sdk-for-java
-                      - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
-                        parameters:
-                          ArtifactName: ${{parameters.ArtifactName}}-${{artifact.name}}-javadevfeed-$(System.JobAttempt)
-                          ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                  - ${{ if eq(parameters.UseEsrpRelease, 'true') }}:
+                    - template: /eng/pipelines/templates/steps/java-publishing.yml
+                      parameters:
+                        ArtifactDirectory: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+                        OutputDirectory: $(Pipeline.Workspace)/EsrpPackages
+                        Target: EsrpRelease
+                        BuildToolsPath: $(Pipeline.Workspace)/azure-sdk-build-tools
+                        JavaRepoRoot: $(Pipeline.Workspace)/azure-sdk-for-java
+                    - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
+                      parameters:
+                        ArtifactName: ${{parameters.ArtifactName}}-esrp-$(System.JobAttempt)
+                        ArtifactPath: $(Pipeline.Workspace)/EsrpPackages
+                  - ${{ else }}:
```

Comment:
Do we have any places where we know we will need this fork? Or is this just in case we need a backup?